yb-voyager/src/migassessment/common.go

```diff
@@ -55,7 +59,12 @@ func LoadCSVDataFile[T any](filePath string) ([]*T, error) {
 
 	for _, record := range records {
 		var tmplRec T
-		bs, err := json.Marshal(record)
+		camelCaseRecord := make(Record)
```

Comment:
Can't we use json tags here? seems wrong to write this logic in a generic `LoadCSVDataFile` function.

Reply:
Json tags needs to be camel case format. We are following that for all our json reports.
But here that is the program since in the database(tablename and column names) are snake case formatted. cc @makalaaneesh

Reply:
Did the changes as discussed. cc @makalaaneesh